packages/compartment-mapper/test/scaffold.js

```diff
@@ -286,21 +286,38 @@ export function scaffold(
         archive = content;
       };
 
+      const sourceMaps = new Set();
+      const sourceMapHook = (sourceMap, { sha512 }) => {
+        sourceMaps.add(sha512);
+        t.log(sha512, sourceMap);
+      };
+
+      const computeSourceMapLocation = ({ sha512 }) => {
+        sourceMaps.delete(sha512);
+        return `${sha512}.map.json`;
+      };
+
       await writeArchive(fakeWrite, readPowers, 'app.agar', fixture, {
         policy,
         modules,
         dev: true,
         tags,
         searchSuffixes,
         commonDependencies,
+        sourceMapHook,
         ...additionalOptions,
       });
+
       const { namespace } = await importArchive(fakeRead, 'app.agar', {
         globals: { ...globals, ...addGlobals },
         modules,
         Compartment,
+        computeSourceMapLocation,
         ...additionalOptions,
       });
+
+      // t.is(sourceMaps.size, 0);
```

Comment:
It happens to pass but frustrates the assertion counting pretty badly, so I stopped here and wrote a different test. I’ll remove the dead code.

Reply:
I’m addressing this by using a thrown exception here instead of frustrating the assertion count (`t.plan`) and getting some coverage over everything using the scaffold.